test/benchmark/lmbench-http-latency/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+echo "*** Running lmbench http latency test ***"
+
+dd if=/dev/zero of=test_file bs=1M count=512
+/benchmark/bin/lmbench/lat_http 127.0.0.1 < test_file
```

Comment:
I don't believe this is a correct usage of `lat_http`. I read the code. The program does not include the step of starting a HTTP server. It seems that there is an another program called `http_srv`, which is the server.

In addition, I doubt the wisdom of providing a file with all zeros.

```c
	while (fgets(file, sizeof(file), stdin)) {
```

From the code, it seems that `lat_http` expects the `stdin` to provide a list of file names.

Before adding a benchmark program to CI, please be sure that you really understand its usage.